packages/editor/src/lib/editor/managers/UserPreferencesManager.ts

```diff
@@ -12,14 +12,16 @@ export class UserPreferencesManager {
 		if (typeof window === 'undefined' || !('matchMedia' in window)) return
 
 		const darkModeMediaQuery = window.matchMedia('(prefers-color-scheme: dark)')
-		if (darkModeMediaQuery?.matches) {
+		if (inferDarkMode && darkModeMediaQuery?.matches) {
 			this.systemColorScheme.set('dark')
 		}
 		darkModeMediaQuery?.addEventListener('change', (e) => {
-			if (e.matches) {
-				this.systemColorScheme.set('dark')
-			} else {
-				this.systemColorScheme.set('light')
+			if (this.getUserPreferences().colorScheme === 'system') {
+				if (e.matches) {
+					this.systemColorScheme.set('dark')
+				} else {
+					this.systemColorScheme.set('light')
+				}
 			}
 		})
```

Comment:
i'm gonna revert these changes, this introduces bugs around switching theme. i.e. if your system is set up to prefer dark and you select 'system' then it won't switch.

I think you already achieved the goal of this PR by just updating the default value to be 'light' instead of 'system'